frontend/src/components/OrderPage.js

```diff
@@ -113,6 +122,44 @@ class OrderPage extends Component {
     this.getOrderDetails(this.state.orderId);
   }
 
+  handleWebln = (data) => {
```

Comment:
The same concept could be applied here, so we can await the `getWebln` function call.